### ISSUE TITLE
Please don't stop developing Omnibar

### ISSUE THREAD

**Issue by vonStein** [rs/SafariOmnibar#88]
I've installed Safari 5.2 and the new original Omnibar isn't half as useful as your Omnibar. So I'm begging you: please keep updating your work. It makes searching so much faster.

Thanks
Martin.

**Comment by rs** [comment]
Which feature from omnibar do you miss in 5.2?

**Comment by vonStein** [comment]
entering "wiki searchterm" opens wikipedia, "ama searchterm" opens amazon and so on. This is something I'm using everyday several times and now every search leads to google. I changed the bundle max identifier so the plugin works with 5.2, it even changes the addressbar input to "Wikipedia: " but this search leads to google again.

And adding custom search engines is a very big plus.

**Comment by rs** [comment]
I miss search engine keywords as well. I think it's possible to implement it thru simple Safari Extension. I'll look at it.

**Comment by vonStein** [comment]
Sounds great! Thanks for all that work! :)

**Comment by dep-deprecated** [comment]
Yes. Custom search engines from the URL bar was a huge loss with Omnibar not being supported in 5.2.  I used Omnibar to do easy queries in to YubNub.org.  From there, I could exact a search on pretty much any website:

CMD-L:
- y goog google search
- y gmap Los Angeles to New York
- y imdb Midnight in Paris
- y netflix Fear and Loathing

etc, etc.   It was low hassle.  So nice.

As a workaround, I've installed Alfred.  A quick launch "spotlight" replacement. With that, I can do the same thing, but through alfred. So instead of CMD-L to focus the searchbar, I do CMD-SPACE to invoke Alfred. Then "y imdb Midnight in Paris" and Safari is opened by default.

You can use Alfred to add any custom search engine you want.  It's a nice workaround, but having this functionality in the browser's URL bar was great. You can do this natively in Firefox by adding a Keyword to a input box. You can do it in Chrome as well under 'edit search engines'

So yes. I highly recommend developing a quick patch on Omnibar for 1.7 to support at least this basic functionality in Safari 5.2. It's huge.

**Comment by dep-deprecated** [comment]
You might also want to check this out:

http://hints.macworld.com/article.php?story=20120213221354176

**Comment by noizo** [comment]
Would love to see that also.

Search engine keywords are just gorgeous.
Please make them live in 5.2 and new 6.0

**Comment by eyalfishler** [comment]
dep - changing the Binary of the browser is never a good idea, more so when you want to have the ability to work with upgrades. we need to get this plugin working with version 6
